tests/e2e/flowtest/flowtest_helpers.py

```diff
@@ -0,0 +1,165 @@
+import logging
```

Comment:
we should try to avoid having helper files. Will it make sense to move this to be under ocs_ci/?

Reply:
There are currently many test cases planned for flow/story based scenarios, so i think a dedicated helper file might be useful. @PrasadDesala @hnallurv What is your opinion on this ?

Reply:
that's understood. What I meant is that we should move the entire file to be placed under ocs_ci/ hierarchy.

Reply:
+1 to move "flowtest_helpers.py " under ocs_ci/

Reply:
ack. Moved